bgpd/bgp_route.c

```diff
@@ -2844,6 +2849,11 @@ static void bgp_process_main_one(struct bgp *bgp, struct bgp_dest *dest,
 			if (bgp_fibupd_safi(safi)
 			    && !bgp_option_check(BGP_OPT_NO_FIB)) {
 
+				if (BGP_SUPPRESS_FIB_ENABLED(bgp)
+				    && new_select->sub_type == BGP_ROUTE_NORMAL)
```

Comment:
Shouldn't this be validated for imported routes as well?

Reply:
imported routes won't have BGP_ROUTE_NORMAL right?